code/core/src/phone/java/com/adobe/marketing/mobile/MobileCore.java

```diff
@@ -82,21 +86,35 @@ public static void setWrapperType(@NonNull final WrapperType wrapperType) {
 
     /**
      * Set the Android {@link Application}, which enables the SDK get the app {@code Context},
-     * register a {@link Application.ActivityLifecycleCallbacks} to monitor the lifecycle of the app
-     * and get the {@link android.app.Activity} on top of the screen.
+     * register a {@link ActivityLifecycleCallbacks} to monitor the lifecycle of the app and get the
+     * {@link Activity} on top of the screen.
      *
      * <p>NOTE: This method should be called right after the app starts, so it gives the SDK all the
      * contexts it needed.
      *
      * @param application the Android {@link Application} instance. It should not be null.
      */
     public static void setApplication(@NonNull final Application application) {
+
         if (application == null) {
             Log.error(
                     CoreConstants.LOG_TAG, LOG_TAG, "setApplication failed - application is null");
             return;
         }
 
+        // Direct boot mode is supported on Android N and above
+        if (VERSION.SDK_INT >= VERSION_CODES.N) {
+            if (UserManagerCompat.isUserUnlocked(application)) {
+                Log.debug(CoreConstants.LOG_TAG, LOG_TAG, "Device is not in direct boot mode.");
+            } else {
+                Log.error(
+                        CoreConstants.LOG_TAG,
+                        LOG_TAG,
+                        "Device is in direct boot mode, SDK will not be initialized.");
```

Comment:
setApplication failed - Device is in direct boot mode, SDK will not be initialized.

Reply:
Can you also update other log statements within this function to be consistent?